src/Endpoints/AxiosConfig.ts

```diff
@@ -16,4 +16,9 @@ axiosInstance.interceptors.response.use((response) => {return response}, (error)
     return Promise.reject(error)
 })
 
+// generic get request to any url provided
+export async function getRequest(url: string): Promise<any> {
```

Comment:
Is this really necessary? It's just a pass-through function

Reply:
Yes. The Tracks[] in the SimplifiedTrackObject only includes a full url to the tracks, meaning we have to make another request just to get the tracks.